Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Dockerfile for building Yarn.
+
+#FROM debian:stretch-slim
```

Comment:
Oops - I was originally trying to use Debian, but I found that some things worked better with Ubuntu for some reason. I'll remove this comment when I get a chance.